app/views/create/next-steps.html

```diff
@@ -89,6 +89,11 @@ <h1 ng-if="!pendingTasks(tasks()).length && erroredTasks(tasks()).length">Comple
               </div>
             </div>
 
+            <div ng-if="templateMessage.length">
+              <h2>Template message</h2>
+              <p>{{templateMessage}}</p>
```

Comment:
I would expect whitespace to be preserved here, so they can have a couple paragraphs. I also think we should probably linkify the text (if they want to point to follow up docs)